play-json/shared/src/main/scala/Writes.scala

```diff
@@ -243,6 +244,20 @@ trait DefaultWrites extends LowPriorityWrites {
   }
 
   /**
+   * Serializer for Scala Duration
+   */
+  implicit val durationWrites: Writes[Duration] = Writes[Duration] {
+    case Duration.Inf => JsString("Inf")
+    case Duration.MinusInf => JsString("MinusInf")
```

Comment:
`Inf` and `MinusInf` are the case name in `Duration`, so for me that's less suprising